lib/web_ui/lib/src/engine/global_styles.dart

```diff
@@ -5,6 +5,7 @@
 import 'browser_detection.dart';
 import 'dom.dart';
 import 'text_editing/text_editing.dart';
+import 'view_embedder/dom_manager.dart';
 
 // Applies the required global CSS to an incoming [DomCSSStyleSheet] `sheet`.
 void applyGlobalCssRulesToSheet(
```

Comment:
This method smells like should be a part of the `StyleManager` bag of methods.

(In fact, should this file just be renamed `style_manager.dart` and put the whole style manager here?)

Reply:
Done!